### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by parulgupta2001**

### CODE REVIEW

File: styles/component-style/login.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  .user-input {
+      width:22rem;
+      border-radius: 0.5rem;
+  }
+
+  .login-btn {
+      background-color: var(--basic-color);
+      color:var(--background-color-one);
+      width:23rem;
+      border-radius: 0.5rem;
```
Comment (Muzammil-khan710) [inline styles/component-style/login.css]: Can this value be converted into px ?

**Review by Muzammil-khan710 — APPROVED** [review]
Code is clean and good. Good class namings used. Good to merge.

**Review by rishabh486 — APPROVED** [review]
good to merge

File: styles/component-style/login.css (hunk 1 above)
Comment (parulgupta2001) [reply styles/component-style/login.css]: yes

**Comment by jDineshKarthic** [comment]
When the user enter the password it should not be visible , try changing the type="password"

**Pull request merged**